app/views/courses/TeacherClassView.coffee

```diff
@@ -294,14 +300,16 @@ module.exports = class TeacherClassView extends RootView
     user = @students.get(userID)
     selectedUsers = new Users([user])
     @enrollStudents(selectedUsers)
-  
+    window.tracker?.trackEvent $(e.currentTarget).data('event-action'), category: 'Teachers', classroomID: @classroom.id, userID: userID, ['Mixpanel']
+
   onClickBulkEnroll: ->
     courseID = @$('.bulk-course-select').val()
     courseInstance = @courseInstances.findWhere({ courseID, classroomID: @classroom.id })
     userIDs = @getSelectedStudentIDs().toArray()
     selectedUsers = new Users(@students.get(userID) for userID in userIDs)
     @enrollStudents(selectedUsers)
-    
+    window.tracker?.trackEvent 'Teachers Class Students Enroll Selected', category: 'Teachers', classroomID: @classroom.id, courseID: courseID, courseInstanceID: courseInstance.id, ['Mixpanel']
```

Comment:
Yea, it goes against our coding guidelines.  I have been leaving them on single lines because they are not usually important to updating or reading the code and I want them to take as little space as possible.

Do you think we should be using multiple lines for this?